src/Tizen.NUI.Wearable/src/public/Popup.cs

```diff
@@ -0,0 +1,568 @@
+                Position = new Position(0, TITLE_POSITION_Y),
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
+                TextColor = Color.Cyan,
+            };
+            Add(title);
+
+            LinearLayout scrollLayout = new LinearLayout();
+            scrollLayout.LinearOrientation = LinearLayout.Orientation.Vertical;
+
+            contentContainer = new View()
+            {
+                Layout = scrollLayout,
+                PositionUsesPivotPoint = true,
+                ParentOrigin = Tizen.NUI.ParentOrigin.TopCenter,
+                PivotPoint = Tizen.NUI.PivotPoint.TopCenter,
+            };
+
+            scroll = new ScrollableBase()
+            {
+                Name = "Scroll",
+                PositionUsesPivotPoint = true,
+                ParentOrigin = Tizen.NUI.ParentOrigin.TopCenter,
+                PivotPoint = Tizen.NUI.PivotPoint.TopCenter,
+                Position = new Position(0, CONTENT_POSITION_Y),
+                ScrollingDirection = ScrollableBase.Direction.Vertical,
+                ScrollEnabled = true,
+                WidthResizePolicy = ResizePolicyType.FillToParent,
+                HeightResizePolicy = ResizePolicyType.FillToParent,
+            };
+            Add(scroll);
+            scroll.Add(contentContainer);
+
+            scrollList = new Dictionary<string, View>();
+
+            TouchEvent += Popup_TouchEvent;
+
+            title.PropertyChanged += Title_PropertyChanged;
+        }
+
+        private void Title_PropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if(e.PropertyName == "Size" || e.PropertyName == "Size2D")
+            {
+                scroll.Position = new Position(0, title.Size.Height);
+            }
+        }
+
+        private static int instanceCnt;
+        private Layer layer;
+        private Dictionary<string, Button> buttonList;
+        private Window window;
+        private TextLabel title;
+        private ScrollableBase scroll;
+        private Dictionary<string, View> scrollList;
+        private EventHandler buttonClick;
+        private Timer timer;
+        private string resourcePath;
+        private PopupStyle popupStyle;
+        private bool wrapContent = false;
+        private View contentContainer;
+
+        private void Iter_ClickEvent(object sender, Button.ClickEventArgs e)
+        {
+            buttonClick?.Invoke(sender, e);
+        }
+        private PointStateType previousState;
+        private bool Popup_TouchEvent(object source, TouchEventArgs e)
+        {
+            if (previousState == PointStateType.Down && e.Touch.GetState(0) == PointStateType.Finished)
+            {
+                OutsideClicked?.Invoke(this, null);
+            }
+            previousState = e.Touch.GetState(0);
+            return true;
+        }
+
+        private void defaultShowAnimation()
```

Comment:
I think target of show/hide animation is not popup.
It should be container of popup because popup can be smaller than screen.

but in tizen wearable case, popup is always cover whole screen so... maybe it works.